src/agent/input-tester/src/logging.rs

```diff
@@ -28,6 +28,7 @@ where
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
 pub struct ProcessDetails<'a> {
```

Comment:
@ranweiler - I'm adding an allow tag here. Clippy thinks this struct is not used, but I see it used in the following function in the same folder, as well as two other files - appverifier.rs and crash_detectors.rs.